src/time_machine.py

```diff
@@ -335,6 +335,13 @@ def time_ns():
             return coordinates_stack[-1].time_ns()
 
 
+def monotonic():
+    return 0.0
+    # if not coordinates_stack:
+    #     return _time_machine.original_time()
+    # return coordinates_stack[-1].time()
```

Comment:
Return 0 on the first request but still keep ticking, unless `tick=False`

This should be done up in `Coordinates` - the calculation in `time_ns()` can be split and `monotonic_ns()` can return `(now_ns - self._real_start_timestamp_ns)` leaving `time_ns` to basically add on the `base` it currently does